scripts/screens/Screens.py

```diff
@@ -536,14 +539,17 @@ def mns_open(self):
             object_id="#text_box_30_horizleft_light",
         )
 
-        if game.clan.current_season == "Newleaf":
-            season_image_id = "#mns_image_newleaf"
-        elif game.clan.current_season == "Greenleaf":
-            season_image_id = "#mns_image_greenleaf"
-        elif game.clan.current_season == "Leaf-bare":
-            season_image_id = "#mns_image_leafbare"
-        elif game.clan.current_season == "Leaf-fall":
-            season_image_id = "#mns_image_leaffall"
+        match game.clan.current_season:
```

Comment:
Unfortunately we can't use match case as we're still supporting windows 7 and match case isn't supported in the version of python that windows 7 uses.